test/clj/cider/nrepl/middleware/util/instrument_test.clj

```diff
@@ -170,3 +170,7 @@
          '#{[(set! (. inst field) (bp (bar) [2])) []] [(bar) [2]]}))
   (is (= (breakpoint-tester '(set! (.field inst) (bar)))
          '#{[(set! (. inst field) (bp (bar) [2])) []] [(bar) [2]]})))
+
+(deftest instrument-deftest
+  (is (= (breakpoint-tester '(deftest foo (bar)))
+         '#{[(bar) [2]]})))
```

Comment:
@grammati Looks like it's not expanding the deftest. Maybe you need to add a `(use 'clojure.test)` inside the test.